packages/nodejs/src/span.ts

```diff
@@ -180,6 +186,15 @@ export class BaseSpan implements Span {
     }
   }
 
+  /**
+   * Returns if the span is still open or not.
+   *
+   * A closed span can't be modified or fetched attributes from.
+   */
+  public isOpen() {
```

Comment:
Why not making this a plain public property?

Reply:
How can I do that without also making it writable? I don't want users to be able to modify it.

The definition `public readonly open: boolean` doesn't work.

```
error TS2540: Cannot assign to 'open' because it is a read-only property.

177     this.open = false
             ~~~~
```

Reply:
I see. I've found a couple of workarounds to make it mutable from the inside but it's probably too much code for a single property. 

LGTM

Reply:
For actual privacy enforced on the JavaScript side, we should use JavaScript private fields, not TypeScript's `private`. We already use those in the `#scopes` and `#roots` properties of the ScopeManager.

It's described as "hard private" in [the TypeScript handbook](https://www.typescriptlang.org/docs/handbook/2/classes.html#caveats):
> Unlike TypeScripts’s `private`, JavaScript’s [private fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) (`#`) remain private after compilation and do not provide the previously mentioned escape hatches like bracket notation access, making them hard private.

Here's [a TypeScript playground](https://www.typescriptlang.org/play?#code/FAYwNghgzlAEAqBPADgUwMogE4EtkBcAFXANwn1QDEdUwATWAb2FldmVPNVgHs0A7AFywARjx5hUEfizYge-KPiwBXEPh5YAFAEomstm3wALHFAB0fVP1gBeWMpWoDsAL7B3oBUtgAzcXaw-KgA7ggoGNh4RJwU1LR0ANzAwAD0qQjG3BxWWPiIsGawSjhgYLAQICCoMDgikgA0QTwUDsbkbUXyALbIpTUOPGkZAAb+PJYCI7w2JtwAUujFOHTO8ooSqOZgPADmWgBEldUwfuKT1gdNWuMVcNKIOhf8OsMV-Aw4+LAg0qLcqDoX0BwESNwC0Hej2egV8EDAUDW3k22z2h2ONTg4xhEF8FCwsG6PCBvkQV1g4J4dyhTysLxS6UKnX4AGtYCFuFBjDwVPRYCpEbBFuxYtxfDR6HAcIoKBA6KBIKd5hAyJhcARiDgyHEJQxmIYAMR04RiTbSFKGdZKVTqTS6fSGQwmMzmI0CQKOZyGTyGZAqeo4ECwXaob50+2MWBYUMqLCzUwWN3WNweFJW74iCAE+zBMLK1VRDWi+L0BkZZ1SrHS7giVC-AXcDmwVbi4IMCDB0P4rwbSSo-ZHKqY0RZ57kzNYZ56MttbjyVb-HZhExYHghe42VBYVcE2v1wVzaMAcjg-CpIfw3YnMPscIRzjecx+xLnPG63WsFAYtaXhXuxUQfh8AgAAPWAtx3Jo9wgBtYBGEJ2goEgt1dEY3jMfgj2+eEl0Bfl+EkU5EB5I9o0KGwO3AaANwYOCEK4ZCsGmIoRgrNDGStFEdn2bEkxeIA) demonstrating the difference.

Reply:
I've updated it.